cylc/flow/batch_sys_handlers/slurm.py

```diff
@@ -51,6 +51,43 @@
 limit known by the suite may be out of sync with what is submitted to the batch
 system.
 
+Cylc supports heterogeneous Slurm jobs via special numbered directive prefixes
+that distinguish repeated directives from one another:
+
+.. code-block:: cylc
+
+   [runtime]
+       # run two heterogenous job components:
+       script = srun sleep 10 : sleep 30
+       [[my_task]]
+           [[[job]]]
+               batch system = slurm
+               execution time limit = PT1H
+           [[[directives]]]
+               --account = QXZ5W2
+               hetjob_0_--mem = 1G  # first prefix must be "0"
+               hetjob_0_--nodes = 3
+               hetjob_1_--mem = 2G
+               hetjob_0_--nodes = 6
```

Comment:
is this deliberate, or should it be 
```suggestion
               hetjob_1_--nodes = 6
```

Reply:
@hjoliver  - can we get this changed or not then we can close?

Reply:
Sorry Tim, I had missed your comment. Applied your change.